projects/plugins/jetpack/modules/search/instant-search/components/search-filters.scss

```diff
@@ -61,6 +61,7 @@
 	.jetpack-instant-search__search-filter-list-label,
 	// Extra specific to override in some themes
 	.widget_search .jetpack-instant-search__search-filter-list-label {
+		color: inherit;
```

Comment:
Nice! `inherit` always feels like the safest option 👍

Reply:
Good use of `inherit` here, which will ensure that the label will inherit the color from `.jetpack-instant-search__search-results-secondary`.